Extensions/xkit_preferences.js

```diff
@@ -1405,33 +1405,41 @@ XKit.extensions.xkit_preferences = new Object({
 
 			$("#xkit-extensions-panel-right-inner").html('<div id="xkit-extension-panel-no-settings">Updating...</div>');
 
-			if (typeof XKit.extensions.xkit_updates === "undefined" || typeof XKit.extensions.xkit_updates.update === "undefined") {
+			if (typeof XKit.extensions.xkit_updates === "undefined" ||
+				typeof XKit.extensions.xkit_updates.update === "undefined") {
+
 				XKit.window.show("Can't update",
-					'It looks like "XKit Updates" extension is missing or not working properly. It is highly recommended that you reset XKit.', "error",
+					'It looks like "XKit Updates" extension is missing or not working properly. '+
```

Comment:
Can we tweak the wording here?  `XKit Updates` is an internal extension that isn't visible through the gallery, I don't think this message would make sense to a user.  Maybe something more general, like "an internal part of XKit is missing or not working properly"?